crates/pathfinder/src/state/sync.rs

```diff
@@ -1049,990 +917,3 @@ pub fn head_poll_interval(chain: Chain) -> std::time::Duration {
         Testnet | Testnet2 | Integration | Custom => Duration::from_secs(30),
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
don't we need to keep some of the basic integration tests to make sure that all necessary messages get through and the head/state is updated properly? otherwise we only would be able to perform a sanity test of those by running a real sync

Reply:
One more refactoring necessary to simplify the tests significantly: https://github.com/eqlabs/pathfinder/discussions/1047